connectors/filtering/basic_rule.py

```diff
@@ -220,6 +232,8 @@ def from_string(cls, string):
 
 
 class BasicRule:
+    """Class representing a basic rule, which is part of the connector's filtering."""
```

Comment:
Same as above - "Class" s redundant, what is this class for, why is it a basic rule and what's not a basic rule?